rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/ast/indicators.kt

```diff
@@ -22,9 +22,7 @@ data class IndicatorExpr(val index: IndicatorKey, override val position: Positio
 @Serializable
 data class GlobalIndicatorExpr(override val position: Position? = null) :
         AssignableExpression(position) {
-    override fun size(): Int {
-        TODO("not implemented") // To change body of created functions use File | Settings | File Templates.
-    }
+    override fun size(): Int = 99
```

Comment:
Replace 99 with the right constant value which is: `IndicatorType.Predefined.range.last`

Reply:
Done